host_core/lib/host_core/control_interface/server.ex

```diff
@@ -50,10 +50,36 @@ defmodule HostCore.ControlInterface.Server do
     Tracer.with_span "Handle Host Ping (ctl)", kind: :server do
       {total, _} = :erlang.statistics(:wall_clock)
 
+      ut_seconds = div(total, 1000)
+
+      ut_human =
+        ut_seconds
+        |> Timex.Duration.from_seconds()
+        |> Timex.Format.Duration.Formatters.Humanized.format()
+
+      {js_domain, ctl_host, prov_rpc_host, rpc_host, lattice_prefix} =
+        case :ets.lookup(:config_table, :config) do
+          [config: config_map] ->
+            {config_map.js_domain, config_map.ctl_host, config_map.prov_rpc_host,
+             config_map.rpc_host, config_map.lattice_prefix}
```

Comment:
Isn't it possible for some of these to be set, but not all? We should have default/fallback values per field

Reply:
The sanitization step ("post process config") ensures that there's always a meaningful value/default for these. So the only time we need to return those other defaults is in the (should never occur) case of config not existing in the ETS table.